### PR TITLE
BaSP-TG: getTaskByDesc and addNewTask endpoints added

### PR DESCRIPTION
**Pull request by Martinzancocchia**

### CODE REVIEW

**Review by paurinaldi — CHANGES_REQUESTED** [review]
Adding to other comments left, I need you to check that the server can run before pushing. There are some slint errors, and the missing [ ] on admins json. Also, tasks json data needs to be on this branch as well as neccesary logic on index, so we can run the methods.

File: src/resources/tasks.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const fileSystem = require('fs');
+const tasks = require('../data/tasks.json');
+
+const router = express.Router();
+
+router.post('/addNewTask', (req, res) => {
+    const newTask = req.body;
+    tasks.push(newTask);
+    fileSystem.writeFile('src/data/tasks.json', JSON.stringify(tasks, null, 4), (err) => {
```
Comment (paurinaldi) [inline src/resources/tasks.js:10]: null and 4 are unnecesary params, please delete

**Review by paurinaldi — CHANGES_REQUESTED** [review]

**Review by paurinaldi — CHANGES_REQUESTED** [review]
Martin, I still can't run the server, check previous comment I left.

File: src/resources/tasks.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const fileSystem = require('fs');
+const tasks = require('../data/tasks.json');
+
+const router = express.Router();
+
+router.post('/addNewTask', (req, res) => {
```
[the remark this answers is not in the document]
Comment (Martinzancocchia) [reply src/resources/tasks.js]: done!

File: src/resources/tasks.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    fileSystem.writeFile('src/data/tasks.json', JSON.stringify(tasks, null, 4), (err) => {
+      if (err) {
+        res.send('Cannot save new task');
+      } else {
+        res.send(`Task added \n ${JSON.stringify(newTask)}`);
+      }
+    });
+  });
+
+  router.get('/getTaskByDesc', (req, res) => {
```
[the remark this answers is not in the document]
Comment (Martinzancocchia) [reply src/resources/tasks.js]: fixed!

File: src/resources/tasks.js (hunk 1 above)
Comment (Martinzancocchia) [reply src/resources/tasks.js]: done!

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**